src/main/ts/ui/Icons.ts

```diff
@@ -0,0 +1,15 @@
+
+  editor.ui.registry.addIcon('smiley', '<svg width="40" height="40" viewBox="0 0 750 750" xmlns="http://www.w3.org/2000/svg"><path d="M347.71 556.03c-30.24-2.25-59.5-13.85-82.89-33.2-15.03-12.44-33.18-20.64-47.44-34.03-41.31-36.41-28.28-97.87-16.8-145.08C219 265.84 271.11 228.29 349.9 225.41c17.33 1.94 61.84-4.72 59.78 21.71-.94 14.43-15.89 12.77-26.28 10.51-11.53-1.61-23.2-2.29-34.83-2.55-38.43-.67-76.84 13.71-105.72 39.01-34.77 39-29.13 97.23-24.53 145.58 13.39 37.54 54.03 58.08 89.35 70.42 57.77 15.24 133.91 28.88 174.95-26.29 26.4-36.41 31.27-83.33 30.59-127.09-2.88-35.13-19.23-70.42-43.24-95.97-16.43-10.66-36.81-12.59-55.16-18.6-6.31-1.82-7.37-10.86-3.42-15.43 6.18-6.01 28.53 5.96 37.3 6.85 33.06 5.95 54.63 32.78 70.88 60.18 32.67 56.79 22.11 131.65-10.46 186.18-36.49 48.86-100.01 81.21-161.41 76.1ZM406.18 140.29c-1.15-11.2.03-22.42.22-33.65 2.88-14-6.77-46.27 12.24-49.5 20.04.72 8.29 50.24 10.25 64.8-.99 10.11 1.41 28.51-12.15 30.32-6.06 0-10.22-6.47-10.57-11.98ZM532.52 213.26c-20.12-33.33 65.54-75.81 89.96-86.55 15.28-4.3 19.07 22.35 6.3 27.49-6.03 2.77-11.98 5.67-17.94 8.57-17.65 8.44-34.24 19.22-48.13 33.07-8.7 6.49-17.65 28.99-30.19 17.42ZM646 353.03c-15.74-2.88-31.94-3.29-47.55-6.54-9.46-2.69-11.44-15.67-6.49-23.16 10.43-12.24 42.35-.59 57.45-.35 14.54 3.34 12.45 31.61-3.41 30.05ZM620 518.14c-11.34-10.19-22.61-20.49-34.57-29.95-11.7-6.96-24.85-23.6-10.44-34.74 14.22-6.37 45.1 30.53 57.55 39.27 11.48 8.59 2.51 35.99-12.54 25.43ZM491.79 634.54c-7.55-18.19-36.14-50.16-15.96-67.17 13.49-8.95 18.38 10.48 22.68 19.61 4.91 9.46 9.87 18.88 15.14 28.15 7.64 12.16-11.65 33.73-21.86 19.4ZM296.06 691.95c-19.08-11.57 1.71-42.49 4.19-59.36 6.55-14.78 6.96-37.01 20.45-46.88 27.69.8 6.08 39.39 2.61 54.06-3.88 11.6-7.41 23.29-10.84 35.04-1.88 8.02-5.33 21.68-16.4 17.14ZM97.3 549.2c-9.18-6.06-8.59-22.57 1.49-27.42 6.01-2.9 12-5.84 17.2-8.38 16.08-7.81 31.71-16.45 47.09-25.56 5.3-3.14 10.51-6.48 16.33-8.59 13.63-3.96 18.14 18.13 8.92 25.29-11.93 10.23-26.63 16.7-40.38 24.13-11.7 4.06-40.95 25.65-50.64 20.53ZM161.03 294.76c-20.58-10.63-39.7-24.15-62.14-31.13-13.44-5.1-9.36-33.33 6.61-28.84 17.87 8.36 77.26 27 68.35 52.42-1.82 5.27-7.28 9.89-12.82 7.56ZM269.71 180.78c-18.98-11.67-36.07-72.16-44.47-94.65-6.27-14.57 12.59-31.05 20.19-12.35 2.2 6.84 4.46 13.66 6.83 20.44 4.9 15.4 11.38 30.13 18.97 44.39 6.47 11.68 21.13 40.12-1.51 42.16Z"/><path d="M300.63 344.21c0-6.1 3.62-14.1 10.72-14.1s10.72 8.01 10.72 14.1-3.62 14.1-10.72 14.1-10.72-8.01-10.72-14.1ZM369.29 321.33c0-6.1 3.62-14.1 10.72-14.1s10.72 8 10.72 14.1-3.62 14.1-10.72 14.1-10.72-8-10.72-14.1ZM351.1 404.51c-16.87-1.4-69.13-1.38-65.61-26.14.52-5.5 4.44-12.12 10.6-12.02 22.38 2.95 44.38 10.32 67.31 8.39 33.6-2.31 42.7-55.23 58.71-38.11 12.11 17.44-10.88 35.09-20.94 47.82-13.34 13.04-31.07 21.91-50.07 20.06Z"/></svg>');
```

Comment:
Is this the correct way to add icons? Might be too long for a single line, there is a limit in `tinymce`